cmd/util/report/report.go

```diff
@@ -0,0 +1,121 @@
+// Copyright (C) 2019-2023 Algorand, Inc.
```

Comment:
mostly former cmd/util/goal/commands.go impls

Reply:
> mostly former cmd/util/goal/commands.go impls

Correct, they had additionally the unicodeprintable check (`printable, line := formatting.UnicodePrintable(line)`), so used them.